test/ipv4or6/src/main.rs

```diff
@@ -24,7 +26,25 @@ where
 
     let pipelines: Vec<_> = ports
         .iter()
-        .map(|port| tcp_nf(ReceiveBatch::new(port.clone()), sched).send(port.clone()))
+        .map(|port| {
```

Comment:
I think we also discussed that for these "small" NB functional tests, we should just use `main.rs` to organize functions and the main call site together. We can revisit separation upon updating the current OG NF.